Docs/development/jasp-build-guide-windows.md

```diff
@@ -0,0 +1,149 @@
+- [RTools](https://cran.r-project.org/bin/windows/Rtools/rtools40.html), for building R modules
+- [Conan](https://github.com/conan-io/conan/releases) > 1.45.0
+- [WIX Toolset](https://wixtoolset.org), if you want to distribute JASP, i.e., creating an installer.
+
+### Installing Visual Studio
+
+Before everything, you need to download and install the Microsoft Visual Studio and make sure that it contains all the necessary C++ libraries and tools. Please follow the steps below:
+
+- Download the Visual Studio with C++ Community Edition 2022 from [Microsoft website](https://visualstudio.microsoft.com/downloads/)
+	- Head to the bottom of the page, select the Windows, and then for the languages and platforms, select the C++. This should give you an installer with all the C++ tools and libraries that we need.
+- If you are installing the VS with C++ Community, you don't need to add or modify your installation, however, if you are downloading the general version, you need to make sure that the following packages are being installed.
+	- During the installation, you will be asked asked to customize your installation, in this section, here, make sure to select the followings, 
+		- From the left panel, you should select the "Desktop development with C++" module. This package includes several tools, e.g.,
+			- Just-In-Time debugger
+			- VC++ 2017
+			- C++ profiling tools
+			- Visual C++ tools for CMake
+
+### Installing Qt Creator and Qt 6
+
+You also need Qt Creator and Qt 6 to be able to build and test JASP's libraries and GUI. For this, 
+
+- You need to download and install the Qt installer from [Qt website](https://www.qt.io/download).
+- After downloading the installer, run the installer and make sure that the following packages are selected for installation
+	- Qt
+		- Qt 6.2.4
+		- Developer and Designer Tools
+			- Qt Creator 7.0 Beta (Recommended)
+			- Qt Creator Debug Symbols
+			- CMake
+			- Ninja
+
+### Installing MSYS2 
+
+Download the MSYS2 from [here](https://www.msys2.org/) and install it in the **default** path, i.e., `C:\msys64`.
+
+> ⚠️ This is important because JASP build system expect to find the MSYS2 in the following default path, otherwise you need to specify your custom path to CMake, using the `MINGW_PATH` variable, e.g., `-DMINGW_PATH=D:\msys64`.
+
+#### Installing MSYS2 Libraries and Packages
+
+After installing MSYS2, you will find a new program in your Start Menu. Search for "MSYS2" in your Start Menu, and from the selection of applications that are showing up, run the one name "MSYS2 MinGW x64". At this point, you should be welcomed with a command prompt.
+
+Copy and paste the following line into the Terminal and press Enter. With this command, we are installing some of required packages and libraries necessary for building JASP.
+
+```bash
+pacman -Syu mingw-w64-x86_64-toolchain mingw-w64-x86_64-boost mingw-w64-x86_64-jsoncpp bison flex mercurial make autoconf
```

Comment:
we don't need `mercurial` anymore right?

Reply:
Let them install it anyway, just kidding! 😅